### DISCUSSION TITLE
Installation Failure

### DISCUSSION THREAD

**Discussion by lbhd** [MilesCranmer/PySR#351] [General]
hello, I tried to install dependencies in my terminal by running: python3 -m pysr install, but run into this issue. Could anyone give me some help on this one? thanks in advance
![1686903324448](https://github.com/MilesCranmer/PySR/assets/54879699/212c60e5-e87a-48f3-b392-2c426b12c558)

**Comment by MilesCranmer** [comment]
It looks like it couldn’t connect to the download servers. Could you try again? Sometimes GitHub goes down.
